cpp/include/cudf/column/column_device_view.cuh

```diff
@@ -39,6 +39,7 @@
 #include <thrust/pair.h>
 
 #include <algorithm>
+#include <functional>
```

Comment:
Where is this used in this file? Also, should it not be `<cuda/std/functional>` ?

Reply:
https://github.com/rapidsai/cudf/blob/ed90437644a0702cb5c4e23ba3e181145a870c04/cpp/include/cudf/column/column_device_view.cuh#L826

There we use `std::function`